copyright

```diff
@@ -0,0 +1,11 @@
+The Orocos RTT library is licensed under the terms of the GNU General
+Public License, with this special exception:
+
+   As a special exception, you may use this file as part of a free software
+   library without restriction.  Specifically, if other files instantiate
+   templates or use macros or inline functions from this file, or you compile
+   this file and link it with other files to produce an executable, this
+   file does not by itself cause the resulting executable to be covered by
+   the GNU General Public License.  This exception does not however
+   invalidate any other reasons why the executable file might be covered by
+   the GNU General Public License.
```

Comment:
There is already a verbose copyright notice in [COPYING](https://github.com/Intermodalics/rtt/blob/7125a6569042a06933817cc26f8b169a5aa0b60c/COPYING), that should be updated instead, or removed.

Reply:
Merged in https://github.com/orocos-toolchain/rtt/pull/264/commits/d4546035a50d7982bc8088f42081c73241e9fa3d.